### PR TITLE
[homepage] add a "by the numbers" section

### PR DESCRIPTION
**Pull request by beets**
Add a section to the homepage highlighting the size of the graph

localhost_8080_ (4)

### CODE REVIEW

File: server/templates/static/homepage.html [hunk 1]
```diff
@@ -90,6 +90,23 @@ <h2>Explore the data</h2>
   </div>
 </section>
 
+<section id="by-the-numbers">
+  <div class="container">
+    <div class="highlight">
+      <h3>By the numbers</h3>
+      <p>We have continuously added data to the Data Commons graph over the last 5 years. As of Jan 2022, the graph contains:</p>
```
Comment (pradh) [inline server/templates/static/homepage.html:97]: Wonder if its 4 years?  Roughly from Aug 2018...

File: server/templates/static/homepage.html [hunk 2]
```diff
@@ -90,6 +90,23 @@ <h2>Explore the data</h2>
...
+      <ul>
+        <li>515 B triples</li>
```
Comment (pradh) [inline server/templates/static/homepage.html:99]: should we expand - B, M, K - to make it clear?

Comment (pradh) [reply server/templates/static/homepage.html:99]: Perhaps `2.8 billion`, `2.8 million`, `90,000`, `200,000` ...

**Review by pradh — APPROVED** [review]
Looks great!

File: server/templates/static/homepage.html (hunk 1 above)
Comment (beets) [reply server/templates/static/homepage.html:97]: thanks for the catch!

File: server/templates/static/homepage.html (hunk 2 above)
Comment (beets) [reply server/templates/static/homepage.html:99]: yea that's a good suggestion!

**Comment by beets** [comment]
thanks for the review -- i'll merge so we can see it on autopush

**Comment by beets** [comment]
good catch! let me send a separate pr for that.

**Pull request merged**